DataManagementSystem/Service/IRODSStorageElementHandler.py

```diff
@@ -440,7 +438,7 @@ def transfer_bulkFromClient( self, fileID, token, ignoredSize, fileHelper ):
     if not os.path.exists( dir_path ):
       return S_ERROR( 'Failed to receive data' )
     try:
-      os.chmod( dir_path, 0755 )
+      os.chmod( dir_path, stat.S_IRWXU | stat.S_IRGRP | stat.S_IXGRP | stat.S_IROTH | stat.S_IXOTH )
```

Comment:
Maybe for human reader it's better if we keep the octal notation as a comment ? But this means we MUST update the comment if we change the actual value :-)